### ISSUE TITLE
lakeshore336 class needs some work

### ISSUE THREAD

**Issue by ambarb** [NSLS-II-SIX/profile_collection#6]
It doesn't play nice with hints
@cmazzoli and @ wenhu, this will likely be an issue for you guys when csx takes the next version of bluesky.

baseline is only recording the readback of the ctrl1 and 2 setpoint .  Maybe this is okay,

bec for 'LiveTable' and `LivePlot` dont work, but the data is recorded in the database.  For now, that is the probe readback in temperature and voltage and the status.

**Comment by awalter-bnl** [comment]
I will need to look at this, I am sure we can resolve the issue by setting appropriate 'kinds'.

**Comment by ambarb** [comment]
@awalter-bnl yes.  I think you are correct.  There is a discussion in the group to move to a common ophyd class so I would not spend tons of time on this right now.  If it is an easy fix for you to make then lets do it.
